libs/CMakeLists.txt

```diff
@@ -8,6 +8,15 @@ if (NPROC EQUAL 0)
   set(NPROC 1)
 endif()
 
+find_package(Boost REQUIRED)
+find_package(BZip2 REQUIRED)
+find_package(Threads REQUIRED)
+set(GFLAGS_NOTHREADS OFF)
+find_package(gflags REQUIRED)
+find_package(fmt)
+find_package(Jemalloc REQUIRED)
+find_package(ZLIB REQUIRED)
```

Comment:
@antonio2368 did you consider specifying versions here?

Reply:
That could introduce some way of verifying that the correct packages are used so I see no reason not to use it.
@antaljanosbenjamin what do you think?
I know `Jemalloc` is custom logic, but everything should be fine.